Week3/homework/step2-7.js

```diff
@@ -20,4 +20,5 @@ f2(y);
 
 console.log(y);
 
-// Add your explanation as a comment here
+// E.g.1 :the number passed to function so that change the argument inside the function only (function's local scope).
+// E.g.2 : changing the argument inside the function affect the property of the object that passed in outside the function so object is deferent.
```

Comment:
The difference is that primitive values (e.g. number, string and booleans) are passed by value and objects (including arrays and functions) are passed by reference.